.gitignore

```diff
@@ -144,3 +144,6 @@ result
 
 # clangd and possibly more
 .cache/
+
+# Mac OS
+.DS_Store
```

Comment:
```suggestion
```

Project-level `.gitignore` is only for project-specific things. Please fix this in your user-level config.

Reply:
Interesting! I didn't even know that existed...